src/uberjob/progress/_simple_progress_observer.py

```diff
@@ -134,15 +132,14 @@ def __init__(
         self._min_update_interval = min_update_interval
         self._max_update_interval = max_update_interval
         self._max_exception_count = max_exception_count
-        self._state = State()
-        self._running_scope_lookup = defaultdict(set)
         self._exception_tuples = []
         self._new_exception_index = 0
         self._lock = threading.Lock()
         self._stale = True
         self._done_event = threading.Event()
         self._thread = None
-        self._start_time = None
+        self._start_time = time.time()
```

Comment:
I set the start time here now instead of __enter__ because it doesn't really make a difference and this is a bit simpler.